editor/src/components/canvas/controls/select-mode/absolute-resize-control.tsx

```diff
@@ -234,13 +238,30 @@ const ResizeEdge = React.memo(
       [maybeClearHighlightsOnHoverEnd],
     )
 
+    const onEdgeDblClick = React.useCallback(
+      (event: React.MouseEvent<HTMLDivElement>) => {
+        if (event.detail != 2) {
```

Comment:
as discussed on discord, this can lead to weird behavior if there are children with position `absolute`, in which case it would make sense to just do nothing here